lang/src/org/partiql/lang/eval/PartiqlAstExtensions.kt

```diff
@@ -0,0 +1,44 @@
+package org.partiql.lang.eval
+
+import com.amazon.ionelement.api.TextElement
+import org.partiql.lang.ast.ExprNode
+import org.partiql.lang.ast.Path
+import org.partiql.lang.ast.VariableReference
+import org.partiql.lang.domains.PartiqlAst
+
```

Comment:
You may have noticed the `@Deprecated` tag added to `lang/src/org/partiql/lang/eval/ExprNodeExtensions.kt` above... This code essentially duplicates that... I would have deleted that file but it's used by a couple of other `AstRewriterBase` implementations that haven't been migrated yet.  We'll have to remember to delete it when they've been migrated.